### ISSUE TITLE
ProgressIndicator A11y Pass

### ISSUE THREAD

**Issue by awalia13** [OfficeDev/office-ui-fabric-react#8388]
Tracking all of the issues brought up with the 03/18 accessibility pass on ProgressIndicator fabric control.Details on individual bugs can be found on[Sharepoint](https://microsoft-my.sharepoint.com/:x:/r/personal/v-amaver_microsoft_com/_layouts/15/Doc.aspx?sourcedoc=%7B31f49823-4e59-4246-b10e-a9d58ed04398%7D&action=default&uid=%7B31F49823-4E59-4246-B10E-A9D58ED04398%7D&ListItemId=1892&ListId=%7B395E1ECB-5002-440E-81C5-6A090E98077C%7D&odsp=1&env=prodbubble)
- [ ] Bug1:[Accessibility][Supporting the Platform]: Current position in progress Indicator is not that much prominent and clear in High contrast modes.
~~- [ ] Bug2:[Accessibility][Screen Readers]: Aria-live notification about beginning and end of the progress event is not announced.~~
- [ ] Bug3:[Accessibility][Usable]: In scan mode backward navigation information about progress bar is announced but focus stays on the previous element only.

**Comment by dzearing** [comment]
Aria-live notifications should be up to the application host; not the progress indicator component. The component has no idea how many indicators will be on the page, and the host should define their own batched aria-live messaging which makes sense for their scenario.

**Comment by 999-Dev** [comment]
Bug3:[Accessibility][Usable]: In scan mode backward navigation information about progress bar is announced but focus stays on the previous element only.

**Note:**
We have logged a new bug in GitHub with Required details and attachment. So we are closing this bug.

New GitHub ID - https://github.com/OfficeDev/office-ui-fabric-react/issues/10792

**Comment by dzearing** [comment]
I want to close this issue once thats separated out.

**Comment by aneeshack4** [comment]
Closing because 1 & 3 have been split into separate bugs.
